watertap/examples/chemistry/tests/test_tutorials.py

```diff
@@ -22,7 +22,7 @@
 import os.path
 from nbconvert.preprocessors import ExecutePreprocessor
 
-_tutorials_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)),'..','..','tutorials')
+_tutorials_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../..', '..', 'tutorials')
```

Comment:
```suggestion
_tutorials_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..','..','..', '..', 'tutorials')
```
That said, this will work on modern Windows systems as-is.

Reply:
Let's open another PR for this, just so we don't need everyone to approve and go through another round of testing.